misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/variable/VariableUtils.java

```diff
@@ -267,4 +269,21 @@ public static String removeRedundantQuotes(String str) {
         } while (str.startsWith("\"") || str.endsWith("\""));
         return str;
     }
+
+    /**
+     * Returns the child variable instance with given name using the parent variable instance.
+     *
+     * @param variable     parent variable
+     * @param childVarName child variable name
+     */
+    public static Value getChildVarByName(BVariable variable, String childVarName) throws Exception {
```

Comment:
Can we throw `EvaluationException, DebugVariableException` instead of generic Exception?
Better if we can add it to the method documentation since we have added it.

Reply:
Thats a good catch. Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/31686/commits/efccb30ed5a03e41512d19dbbc53def42a94b814. Thanks for pointing out